source/reference/pinning-nixpkgs.md

```diff
@@ -1,35 +0,0 @@
-(ref-pinning-nixpkgs)=
-
-# Pinning Nixpkgs
```

Comment:
We should not remove that without replacement, yet. I don't know of any good resource where the multiple ways specifying inputs without flakes is documented in one place.

Reply:
I think https://nix.dev/tutorials/towards-reproducibility-pinning-nixpkgs is a better article on this.

Not sure if we should add a section on `$NIX_PATH` however since I feel like that's not something we want someone to do.

Reply:
Though at least explaining how `import <nixpkgs>` magically resolves to a `nixpkgs` might be a good idea. Just not sure where to put it.

I guess "towards reproducibility pinning nixpkgs" can have an intro showing what the problem with that first so the reader understands why it's an issue in the first place.

Reply:
Yeah, just listing that same at the right spot is fine, and that article is probably a good place.